pkg/services/featuremgmt/registry.go

```diff
@@ -2014,8 +2031,8 @@ var (
 		},
 		{
 			Name:         "dashboardTemplates",
-			Description:  "Enable template dashboards",
-			Stage:        FeatureStageExperimental,
+			Description:  "Suggests template dashboards when creating new dashboards",
```

Comment:
The word `suggest` is a bit confusing here since we have a `suggestedDashboards` FF too. Can we be more explicit in both descriptions?